browser-interface/packages/shared/comms/sceneSubscriptions.ts

```diff
@@ -14,18 +14,3 @@ export function subscribeParcelSceneToCommsMessages(controller: ICommunicationsC
 export function unsubscribeParcelSceneToCommsMessages(controller: ICommunicationsController) {
   scenesSubscribedToCommsEvents.delete(controller)
 }
-
-/**
- * Retrieve the scene IDs that are subscribed to receive scene messages from comms
- *
- * @returns a list of CIDs, identifying the Scenes by their hash
- */
-export function getParcelSceneSubscriptions(): string[] {
```

Comment:
Not used? This is supposed to be used to make the MessageBus work

Reply:
https://sourcegraph.com/search?q=context%3A%40eordano%2Fdecentraland+getParcelSceneSubscriptions&patternType=standard&sm=1&groupBy=repo